### PR TITLE
Windows Behavior Adapted

### PR DESCRIPTION
**Pull request by Venomalia**
Metadata, Emulated and Host Devices Windows appear in a position relative to the main window and no longer change their size on their own.
asks the user if he really wants to quit if he has unsaved work.
The version number is now displayed in the main window.

### CODE REVIEW

File: MainWindow.xaml.cs [hunk 1]
```diff
@@ -270,5 +268,54 @@ private void ButtonFill_Click(object sender, RoutedEventArgs e)
         {
             PanZoom.ViewModel.FillRegion();
         }
+
+        #region Closing
+
+        public bool UnsavedChanges
```
Comment (iwubcode) [inline MainWindow.xaml.cs:274]: This property seems odd to me.

I'd assume we *always* want to listen to the `CollectionChanged` events.  And anytime a change occurs, we set `unsavedChanges` to true.  Why are we removing the `ChangesObserved` event if we set to true?

File: MainWindow.xaml.cs [hunk 2]
```diff
@@ -270,5 +268,54 @@ private void ButtonFill_Click(object sender, RoutedEventArgs e)
...
+            {
+                MessageBoxResult MessageResult;
+                MessageResult = MessageBox.Show("unsaved changes are lost, do you want to save?", "unsaved changes!", MessageBoxButton.YesNoCancel);
+                switch (MessageResult)
+                {
+                    case MessageBoxResult.Cancel:
+                        e.Cancel = true;
+                        break;
+                    case MessageBoxResult.Yes:
+                        SaveAsData_Click(sender, new RoutedEventArgs());
```
Comment (iwubcode) [inline MainWindow.xaml.cs:311]: I think this should be `SaveData_Click` if the document has been saved already, otherwise `SaveAsData_Click`

Comment (iwubcode) [reply MainWindow.xaml.cs:311]: Looks like just changing this to `SaveData_Click` is good enough

File: MainWindow.xaml.cs (hunk 1 above)
Comment (Venomalia) [reply MainWindow.xaml.cs:274]: we could do that, I just prefer not to have requests coming in all the time when they are not necessary.

File: MainWindow.xaml.cs (hunk 2 above)
Comment (Venomalia) [reply MainWindow.xaml.cs]: that is right, perhaps most logical for the user.

File: MainWindow.xaml.cs (hunk 1 above)
Comment (iwubcode) [reply MainWindow.xaml.cs:274]: Ah, I was typing out the steps to ask a question about it and now I see how it works.  Clever.

**Pull request merged**